### ISSUE TITLE
Questions about training speed and memory usage when finetuning the model.

### ISSUE THREAD

**Issue by insoo-best** [FoundationVision/Infinity#116]
I have been working with the Infinity model and encountered some issues during fine-tuning. I would greatly appreciate your insights on the following:

1. When performing full fine-tuning of the Infinity model, the training speed appears to be significantly slow, compared with diffusion models. Could you advise on which parts of the model might be causing this slowdown?

2. Interestingly, when I fine-tune only a subset of the parameters (as opposed to the full model), the training speed remains roughly the same as with full fine-tuning. Typically, I would expect faster training when fewer parameters are updated. Could you explain why this happens?

3. Lastly, when I fine-tune a subset of the parameters, I expect I can increase a batch size, compared with full fine-tuning. However, increasing the batch size results in an out-of-memory (OOM) error. Why might this occur, even though fewer parameters are being trained?

Thank you in advance for your support.

**Comment by ZeroRF** [comment]
Hi, what accuracy (accm) can be achieved when fine-tuning Infinity? I've tried many different settings and can only get around 70.

**Comment by smileyenot983** [comment]
> Hi, what accuracy (accm) can be achieved when fine-tuning Infinity? I've tried many different settings and can only get around 70.

Hi, do you have any sample images for accm = 70?
